lib/stellar/client.rb

```diff
@@ -108,25 +109,33 @@ def create_account(options={})
       amount:   Stellar::Amount
     }) => Any
     def send_payment(options={})
-      from     = options[:from]
-      sequence = options[:sequence] || (account_info(from).sequence.to_i + 1)
+      from_account     = options[:from]
+      channel_account = options[:channel_account]
```

Comment:
Can we change `channel_account` to, simply, `transaction_source`. I think that name is less confusing.